docs/resources/cloud_provider_aws_cloudwatch_scrape_job.md

```diff
@@ -82,6 +92,7 @@ resource "grafana_cloud_provider_aws_cloudwatch_scrape_job" "test" {
 - `export_tags` (Boolean) When enabled, AWS resource tags are exported as Prometheus labels to metrics formatted as `aws_<service_name>_info`.
 - `regions_subset_override` (Set of String) A subset of the regions that are configured in the associated AWS Account resource to apply to this scrape job. If not set or empty, all of the Account resource's regions are scraped.
 - `service` (Block List) One or more configuration blocks to configure AWS services for the CloudWatch Scrape Job to scrape. Each block must have a distinct `name` attribute. When accessing this as an attribute reference, it is a list of objects. (see [below for nested schema](#nestedblock--service))
+- `static_labels` (Block List) Zero or more configuration blocks to configure static labels to add to all metrics exported by this scrape job. (see [below for nested schema](#nestedblock--static_labels))
```

Comment:
I think the name of the field needs to be `static_label` to match the TF config, going on what's in the examples. Same for the data source.

Reply:
@tristanburgess, you are right 👍 Nice catch!